features/rails_features/breadcrumbs.feature

```diff
@@ -0,0 +1,106 @@
+  And the event "breadcrumbs.0.timestamp" is a timestamp
+  And the event "breadcrumbs.0.metaData.controller" equals "BreadcrumbsController"
+  And the event "breadcrumbs.0.metaData.action" equals "handled"
+  And the event "breadcrumbs.0.metaData.method" equals "GET"
+  And the event "breadcrumbs.0.metaData.path" equals "/breadcrumbs/handled"
+  And the event "breadcrumbs.0.metaData.event_name" equals "start_processing.action_controller"
+
+  Examples:
+    | ruby_version | rails_version |
+    | 2.0          | 3             |
+    | 2.1          | 3             |
+    | 2.2          | 3             |
+    | 2.2          | 4             |
+    | 2.2          | 5             |
+    | 2.3          | 3             |
+    | 2.3          | 4             |
+    | 2.3          | 5             |
+    | 2.4          | 3             |
+    | 2.4          | 5             |
+    | 2.5          | 3             |
+    | 2.5          | 5             |
+
+Scenario Outline: SQL Breadcrumb
+  Given I set environment variable "RUBY_VERSION" to "<ruby_version>"
+  And I start the service "rails<rails_version>"
+  And I wait for the app to respond on port "6128<rails_version>"
+  When I navigate to the route "/breadcrumbs/sql_breadcrumb" on port "6128<rails_version>"
+  Then I should receive a request
+  And the request is a valid for the error reporting API
+  And the request used the "Ruby Bugsnag Notifier" notifier
+  And the request contained the api key "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And the event has a "process" breadcrumb named "ActiveRecord SQL query"
+
+  Examples:
+    | ruby_version | rails_version |
+    | 2.0          | 3             |
+    | 2.1          | 3             |
+    | 2.2          | 3             |
+    | 2.2          | 4             |
+    | 2.2          | 5             |
+    | 2.3          | 3             |
+    | 2.3          | 4             |
+    | 2.3          | 5             |
+    | 2.4          | 3             |
+    | 2.4          | 5             |
+    | 2.5          | 3             |
+    | 2.5          | 5             |
+
+Scenario Outline: Active job breadcrumb
+  Given I set environment variable "RUBY_VERSION" to "<ruby_version>"
+  And I start the service "rails<rails_version>"
+  And I wait for the app to respond on port "6128<rails_version>"
+  When I navigate to the route "/breadcrumbs/active_job" on port "6128<rails_version>"
+  Then I should receive a request
+  And the request is a valid for the error reporting API
+  And the request used the "Ruby Bugsnag Notifier" notifier
+  And the request contained the api key "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And the event has a "process" breadcrumb named "Start perform ActiveJob"
+  And the event "breadcrumbs.0.timestamp" is a timestamp
+  And the event "breadcrumbs.0.metaData.event_name" equals "perform_start.active_job"
+
+  Examples:
+    | ruby_version | rails_version |
+    | 2.2          | 4             |
```

Comment:
Lines 81 and 83 are failing for me with:
```
 2.2          | 4             |
      <"perform_start.active_job"> expected but was
      <"start_processing.action_controller">. (Test::Unit::AssertionFailedError)
      features/rails_features/breadcrumbs.feature:81:in `And the event "breadcrumbs.0.metaData.event_name" equals "perform_start.active_job"'
      features/rails_features/breadcrumbs.feature:77:in `And the event "breadcrumbs.0.metaData.event_name" equals "perform_start.active_job"'

Failing Scenarios:
cucumber features/rails_features/breadcrumbs.feature:81 # Scenario Outline: Active job breadcrumb, Examples (#1)
```

Reply:
Just checked after the latest update, and these tests are now passing ✅